interface/models.py

```diff
@@ -89,7 +92,7 @@ def get_url(self):
         return storage.get_link(f'{self.id}.zip')
 
     @property
```

Comment:
You should remove this line. This is a fully fledged function, not a property. Plus by leaving `@property` it isn't callable anymore.